REQUIRE

```diff
@@ -2,3 +2,4 @@ julia 0.3
 DataFrames
 Reexport
 Compat
+RData
```

Comment:
RData doesn't support Julia 0.3, so you'll have to bump the Julia requirement in REQUIRE as well.